src/App.tsx

```diff
@@ -21,6 +29,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={event => {
+              setItemsPerPage(+event.target.value);
+              setPage(1);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)